packages/perseus/src/widgets/interactive-graphs/graphs/linear-system.tsx

```diff
@@ -56,7 +120,82 @@ const LinearSystemGraph = (props: LinearSystemGraphProps) => {
                     color="var(--movable-line-stroke-color)"
                 />
             ))}
-            ;
-        </>
+            {linesAriaInfo.map(
+                ({
+                    pointsDescriptionId,
+                    interceptDescriptionId,
+                    slopeDescriptionId,
+                    pointsDescription,
+                    interceptDescription,
+                    slopeDescription,
+                }) => (
+                    <>
+                        <g
+                            key={pointsDescriptionId}
+                            id={pointsDescriptionId}
+                            style={{display: "hidden"}}
+                        >
+                            {pointsDescription}
+                        </g>
+                        <g
+                            key={interceptDescriptionId}
+                            id={interceptDescriptionId}
+                            style={{display: "hidden"}}
+                        >
+                            {interceptDescription}
+                        </g>
+                        <g
+                            key={slopeDescriptionId}
+                            id={slopeDescriptionId}
+                            style={{display: "hidden"}}
```

Comment:
What is the purpose of `display: hidden` here? I don't think `hidden` is a valid value for `display` ([MDN docs](https://developer.mozilla.org/en-US/docs/Web/CSS/display)) and either `display: none` or `visibility: hidden` [will hide the content from screenreaders](https://developer.mozilla.org/en-US/docs/Web/CSS/visibility#accessibility).

I think we probably want to apply an `srOnly` set of styles here.

Reply:
omg

Thank you for pointing this out!

Reply:
I think this also means we can use `innerText` in the SR tree like you suggested rather than `textContent`.